src/main/java/org/javarosa/core/model/SelectChoice.java

```diff
@@ -10,31 +15,51 @@
 import org.javarosa.core.util.externalizable.PrototypeFactory;
 import org.javarosa.xform.parse.XFormParseException;
 
-import java.io.DataInputStream;
-import java.io.DataOutputStream;
-import java.io.IOException;
-
 public class SelectChoice implements Externalizable, Localizable {
+    /** The literal value from the child that is used as a value. For inline selects ("static"), this child always has
```

Comment:
As I wrote in the commit message, this structure is really confusing. Really, there are multiple types of choices (localized dynamic, non-localized dynamic, localized static, non-localized static). I wrote the comments to untangle myself and I think they can be helpful for an eventual refactor.

Reply:
I don't disagree with you. 😅 I added them in because in the code they're currently only described as static vs dynamic as far as I can tell. Part of the reason I wrote these comments is because you had mentioned static vs dynamic coming from the Collect side of things and I wasn't totally sure that it was inline vs itemset.

Can remove but wanted to clarify that it provides value to me and connects this to other parts of the code.